catboost/python-package/ut/medium/test.py

```diff
@@ -1841,6 +1841,18 @@ def test_shap_feature_importance(task_type):
     return local_canonical_file(fimp_npy_path)
 
 
+def test_shap_feature_importance_modes(task_type):
+    pool = Pool(TRAIN_FILE, column_description=CD_FILE)
+    model = CatBoostClassifier(iterations=5, learning_rate=0.03, max_ctr_complexity=1, task_type=task_type, devices='0')
```

Comment:
I suggest to not set max_ctr_complexity to 1, because we need this to work with combinations also.